components/treeview/stories/treeview.stories.js

```diff
@@ -0,0 +1,241 @@
+// Import the component markup template
+import { Template } from "./template";
+
+export default {
+  title: "Treeview",
+  description: "The typical usage of a treeview involves nesting a .spectrum-Treeview element within the .spectrum-TreeView-item parent element.",
+  component: "Treeview",
+  argTypes: {
+    items: { table: { disable: true } },
+    size: {
+      name: "Size",
+      type: { name: "string", required: true },
+      table: {
+        type: { summary: "string" },
+        category: "Component",
+      },
+      options: ["s", "m", "l", "xl"],
+      control: "select"
+    },
+    isQuiet: {
```

Comment:
Could you add `description` properties to the control(s)? Especially for `isQuiet`, since its effect may not be obvious.

Reply:
It's a common styling type for Spectrum and we haven't been adding descriptions to controls really... I honestly don't know what the proper description for it would be. It makes the styling the Quiet type? I think it should be alright but I'll ask @castastrophe

Reply:
Yeah we can leave it off for now. What I'd love to do eventually is create a standard set of common controls and have every component just import that. Once we migrate to using schemas to populate the controls, that will be much easier. This look great for now!